### PR TITLE
chore: document how to import components from local repo

### PR DESCRIPTION
**Pull request by vursen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+      config.server.watch.ignored = [`!${nodeModules}/**`];
+      config.optimizeDeps ??= {};
+      config.optimizeDeps.exclude = [
+        ...(config.optimizeDeps.exclude ?? []),
+        '@vaadin/react-components',
+        '@vaadin/react-components-pro',
+      ];
+    },
+    resolveId(id) {
+      if (/^(@vaadin\/react-components)/.test(id)) {
```
Comment (vursen) [inline README.md:29]:
```suggestion
      if (/^(@vaadin|@polymer)/.test(id)) {
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
 # Vaadin React components
 
 React wrappers for Vaadin components.
+
+## Using Local React Components in a Vaadin Project
+
+To configure your Vaadin project to import React components from a local repository, add this Vite plugin to `vite.config.ts`:
```
Comment (tomivirkki) [inline README.md:7]: Why not add the plugin to `vite.config.ts` in this PR?

Comment (vursen) [reply README.md:7]: I'm not sure what you mean... This plugin is intended to be used in other projects, not react-components itself.

Comment (vursen) [inline README.md:7]:
```suggestion
To configure your Vaadin project to import React components from a local repository, add this Vite plugin to the project's `vite.config.ts`:
```

Comment (tomivirkki) [reply README.md]: Ah, got it. Misunderstood the purpose.

Comment (vursen) [reply README.md]: I updated the text to clarify this.

**Review by sissbruecker — APPROVED** [review]

**Pull request merged**